ci/testnet-manager.sh

```diff
@@ -102,79 +130,77 @@ steps:
       env:
         TESTNET: "$TESTNET"
         TESTNET_OP: "$TESTNET_OP"
+        TESTNET_DB_HOST: "$TESTNET_DB_HOST"
+        EC2_NODE_COUNT: "$EC2_NODE_COUNT"
+        GCE_NODE_COUNT: "$GCE_NODE_COUNT"
 EOF
     ) | buildkite-agent pipeline upload
     exit 0
   fi
 fi
 
-
 sanity() {
   echo "--- sanity $TESTNET"
   case $TESTNET in
   testnet-edge)
-    # shellcheck disable=2030
-    # shellcheck disable=2031
     (
-      set -ex
-      export NO_LEDGER_VERIFY=1
-      export NO_VALIDATOR_SANITY=1
-      ci/testnet-sanity.sh edge-testnet-solana-com ec2 us-west-1a
+      set -x
+      NO_LEDGER_VERIFY=1 \
+        ci/testnet-sanity.sh edge-testnet-solana-com ec2 us-west-1a
     )
     ;;
   testnet-edge-perf)
-    # shellcheck disable=2030
-    # shellcheck disable=2031
     (
-      set -ex
-      export REJECT_EXTRA_NODES=1
-      export NO_LEDGER_VERIFY=1
-      export NO_VALIDATOR_SANITY=1
-      ci/testnet-sanity.sh edge-perf-testnet-solana-com ec2 us-west-2b
+      set -x
+      REJECT_EXTRA_NODES=1 \
+      NO_LEDGER_VERIFY=1 \
+      NO_VALIDATOR_SANITY=1 \
+        ci/testnet-sanity.sh edge-perf-testnet-solana-com ec2 us-west-2b
     )
     ;;
   testnet-beta)
-    # shellcheck disable=2030
-    # shellcheck disable=2031
     (
-      set -ex
-      export NO_LEDGER_VERIFY=1
-      export NO_VALIDATOR_SANITY=1
-      ci/testnet-sanity.sh beta-testnet-solana-com ec2 us-west-1a
+      set -x
+
+      ok=true
+      if [[ -n $EC2_NODE_COUNT ]]; then
+        NO_LEDGER_VERIFY=1 \
+          ci/testnet-sanity.sh beta-testnet-solana-com ec2 "${EC2_ZONES[0]}" || ok=false
+      elif [[ -n $GCE_NODE_COUNT ]]; then
+        NO_LEDGER_VERIFY=1 \
+          ci/testnet-sanity.sh beta-testnet-solana-com gce "${GCE_ZONES[0]}" || ok=false
+      else
+        echo "Error: no EC2 or GCE nodes"
+        ok=false
+      fi
+      $ok
     )
     ;;
   testnet-beta-perf)
-    # shellcheck disable=2030
-    # shellcheck disable=2031
     (
-      set -ex
-      export REJECT_EXTRA_NODES=1
-      export NO_LEDGER_VERIFY=1
-      export NO_VALIDATOR_SANITY=1
-      ci/testnet-sanity.sh beta-perf-testnet-solana-com ec2 us-west-2b
+      set -x
+      REJECT_EXTRA_NODES=1 \
+      NO_LEDGER_VERIFY=1 \
+      NO_VALIDATOR_SANITY=1 \
+        ci/testnet-sanity.sh beta-perf-testnet-solana-com ec2 us-west-2b
     )
     ;;
   testnet)
-    # shellcheck disable=2030
-    # shellcheck disable=2031
     (
-      set -ex
-      export NO_LEDGER_VERIFY=1
-      export NO_VALIDATOR_SANITY=1
```

Comment:
Any reason to not include NO_VALIDATOR_SANITY flag in this deployment?

Reply:
imo not worth the risk for v0.11

Reply:
Should I commit it as shown then?  This file is just an exact copy of the one on master at the moment

Reply:
I think we should add it back, at least till tesnet is on v11